mmv1/products/identityplatform/ProjectDefaultConfig.yaml

```diff
@@ -124,3 +124,136 @@ properties:
             output: true
             description: |
               Memory cost for hash calculation. Used by scrypt and other similar password derivation algorithms. See https://tools.ietf.org/html/rfc7914 for explanation of field.
+
+  - !ruby/object:Api::Type::NestedObject
+    name: 'blockingFunctions'
+    description: |
+      Configuration related to blocking functions.
+    properties:
+      - !ruby/object:Api::Type::Map
+        name: 'triggers'
+        description: |
+          Map of Trigger to event type. Key should be one of the supported event types: "beforeCreate", "beforeSignIn".
+        value_type: !ruby/object:Api::Type::NestedObject
+          properties:
+            - !ruby/object:Api::Type::String
+              name: 'functionUri'
```

Comment:
would it make sense for this to be a resource reference to https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/cloudfunctions_function instead of a pure string?  

In theory we could accommodate someone that uses Terraform for Auth but not Functions, but do we expect that to be a common use-case?

Reply:
Would defer this to the reviewer to weigh in.

Reply:
Yep. Done.